perf/load/templates/config-map.yaml

```diff
@@ -46,7 +46,8 @@ data:
     export INGRESS_IP={{ .Values.ingress }}
 
     ${WD}/rollout.sh &
-    ${WD}/rotate.sh {{ .Values.serviceNamePrefix }}0 {{ .Values.domain }} 60m &
+    ${WD}/scaling.sh &
+    {{/*${WD}/rotate.sh {{ .Values.serviceNamePrefix }}0 {{ .Values.domain }} 60m &*/}}
```

Comment:
what is `/*  .... */` doing here?

Reply:
an accident, let me fix this

Reply:
Ping.
I think this is great to get in quickly.

Reply:
Fixed